druid-shell/src/platform/gtk/window.rs

```diff
@@ -1209,23 +1209,19 @@ impl IdleHandle {
     {
         let mut queue = self.idle_queue.lock().unwrap();
         if let Some(state) = self.state.upgrade() {
+            queue.push(IdleKind::Callback(Box::new(callback)));
```

Comment:
This won't work, because after you push, the queue won't be empty...

Reply:
:man_facepalming: `clippy --fix` failed me. Good catch  :)